scripts/compute_average_distance.py

```
@@ -0,0 +1,547 @@
+        seg_dist = []
+        failed = 0
+        for i in range(len(coords) - 1):
+            sd = straight[i]
+            rd = route_anchors(
+                adj, snapped[i], snapped[i + 1], max(sd * MAX_RATIO, sd + 5000)
+            )
+            if rd is None or rd > sd * MAX_RATIO or rd < sd * MIN_RATIO:
+                failed += 1
+                seg_dist.append(sd)  # 失敗区間は直線距離で代替
+            else:
+                # 実距離は直線距離を下回らない（駅は線路から横にずれているため
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)